src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
```

Comment:
The message printed should be more informative and indicate that it is an Excavator that is working. Consider including the type of machine in the message, for example: 'Excavator is starting its work'.